assets/a1111webui.js

```diff
@@ -92,21 +92,19 @@ function crunchParamHook(data, key, value) {
     data['negativeprompt'] = data['negativeprompt'].replaceAll(match, replace);
 }
 
-function crunchMetadata(url) {
+// Public function
```

Comment:
... what?

Reply:
Simply making obvious those are accessed outside of the scope of the file.